docs/source/whitepaper.rst

```diff
@@ -322,10 +322,9 @@ In any case, this condition will always be fulfilled if Bob fills the request be
 Self challenges
 +++++++++++++++
 
-To make the protocol easier to reason about and implement, only two actors can participate in a challenge: the original
-claimer, and the initial challenger. This raises the concern that, after submitting his false claim, Charles could challenge
-himself to prevent anyone from challenging him. This would let Charlie control the state of his challenge and he would be able to
-let it expire with his claim successful as an outcome.
+In the previous explanation, we only talked about two participants to the challenges, Bob and Charles. What if, after
+submitting his false claim, Charles challenges himself. This would let Charles control the state of his challenge and
+he would be able to let it expire with his claim successful as an outcome.
 
 To prevent this successful `self-challenged claim` to allow Charles to withdraw Alice's deposit, Bob can fill Alice's request
```

Comment:
```suggestion
To prevent this successful `self-challenged claim` allowing Charles to withdraw Alice's deposit, Bob can fill Alice's request
```

Reply:
What you want to prevent is the allowance of Charles to withdraw, not the claim. So `to allow` is correct.

Reply:
I did not know that it changes meaning if you change from to+infinitive to a gerund. But I'm certainly not a grammar expert.
The sentence still sounds weird to me